src/DependencyInjection/Configuration.php

```diff
@@ -54,12 +54,7 @@ public function __construct($debug)
     public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder('httplug');
-        // Keep compatibility with symfony/config < 4.2
-        if (!method_exists($treeBuilder, 'getRootNode')) {
-            $rootNode = $treeBuilder->root('httplug');
-        } else {
-            $rootNode = $treeBuilder->getRootNode();
-        }
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
has this been backported to symfony 3.4?

Reply:
oops

Reply:
reverted